Readme.md

```diff
@@ -2,7 +2,10 @@ istanbul-combine
 ================
 Combine coverage reports from multiple runs in to a single coverage report.
 
+[![Npm Version](https://img.shields.io/npm/v/istanbul-combine.svg)](https://npmjs.com/package/istanbul-combine)
 [![Build Status](https://travis-ci.org/jamestalmage/istanbul-combine.svg?branch=master)](https://travis-ci.org/jamestalmage/istanbul-combine)
+[![Coverage Status](https://coveralls.io/repos/jamestalmage/istanbul-combine/badge.svg?branch=master&service=github)](https://coveralls.io/github/jamestalmage/istanbul-combine?branch=master)
```

Comment:
Coverage is meaningless on this project. This would only report the coverage for the fixtures.